### PR TITLE
CODE-90 enable eslint standard and react plugins

### PR DESCRIPTION
**Pull request by NagaPradeepGajula-osi**
This PR enables the eslint standard rules and react plugin specifying the best practices. Enabling this  brought up 245 errors in the code base. Most them include react/prop-types, react/display-name, no-unused-vars.

### CODE REVIEW

File: apps/www/package.json [hunk 1]
```diff
@@ -23,9 +23,6 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject"
   },
-  "eslintConfig": {
```
Comment (eckdanny-osi) [inline apps/www/package.json]: this is CRA boilerplate. I think it should be `"extends": "<path to our .eslintrc.js>"`

Comment (eckdanny-osi) [reply apps/www/package.json]: CRA pumps lint warnings etc to terminal window bound to the react-scripts process

**Comment by eckdanny-osi** [comment]
I have mixed feelings re any propType or displayName... will have to pull down and play some.

File: apps/www/package.json (hunk 1 above)
Comment (NagaPradeepGajula-osi) [reply apps/www/package.json]: Ah! nice point @eckdanny-osi I will get to it.

File: .eslintrc.js [hunk 2]
```diff
@@ -1,13 +1,18 @@
 module.exports = {
   extends: [
+    'standard',
+    'plugin:react/recommended',
     'prettier',
+    'prettier/react',
+    'prettier/standard'
   ],
-  plugins: ['prettier'],
+  plugins: ['react', 'prettier', 'standard', 'jest'],
```
Comment (eckdanny-osi) [inline .eslintrc.js]: does order matter here like in `extends`<sup>[docs](https://github.com/prettier/eslint-config-prettier#installation)</sup> ('_...make sure to put it **last**_') ?

I don't _think_ so, but idunno. Pretty sure what you have is 👍

Comment (NagaPradeepGajula-osi) [reply .eslintrc.js]: order doesn't matter.

File: apps/www/package.json (hunk 1 above)
Comment (eckdanny-osi) [reply apps/www/package.json]: err.... I don't know if this is possible. 

Deep in `react-scripts`, the pretty terminal output & browser overlay lint errors are hardcoded: [relevant section of src](https://github.com/facebook/create-react-app/blob/6e14dbbbb8a540076c2315a8664dcc6405f5f8e2/packages/react-scripts/config/webpack.config.dev.js#L190)

**Comment by eckdanny-osi** [comment]
@NagaPradeepGajula-osi badass. 

* [x] Format on Save
* [x] Desired DX in `www` 
* [x] editor integration

Re rules,

| rule | count | |
| :--- | :---: | --- |
| react/prop-types | 92 | 👍 |
| no-unused-vars | 85 | 👍 |
| react/display-name | 22 | 👎 |

I dunno if display name is worth caring about. When we had more HOCs and recompose stuff going on, i would prefer not to include the rule. If the rest of the program is firmly _for_ setting `displayName`, then I suppose this monorepo should follow suit for consistency's sake. I bet there's a code-mod for displayName, or it'd be a fine _help wanted_ task for a new contributor unfamiliar with the code to take on.

**Comment by NagaPradeepGajula-osi** [comment]
react/displayName we can disable it if wanted. I am also not stringent on it.

**Pull request merged**